cli/config.json

```diff
@@ -1,10 +1,11 @@
 {
   "rpcUrl": "http://localhost:3332",
-  "rpcRecipient": "AV3LnV78ezsEBZebNeMPtEcH1hmvSfUBC5Xbyrzqbt44",
+  "rpcRecipient": "DJ8bJdeumiffGMmRW7WpfBXk4HkBjkLuJ9y2n63A72at",
   "solanaRpcUrl": "http://127.0.0.1:8899",
   "rpcPublicKey": "EkXDLi1APzu6oxJbg5Hnjb24kfKauJp1xCb5FAUMxf9D",
   "lookupTable": "8SezKuv7wMNPd574Sq4rQ1wvVrxa22xPYtkeruJRjrhG",
```

Comment:
@SwenSchaeferjohann Question for you here. I guess that this diff comes from renaming `lookupTable` <-> `lookUpTable`. I think the former (`lookupTable`) is more correct ([Wikipedia says so too](https://en.wikipedia.org/wiki/Lookup_table)), but just trying to make sure. How could we adjust this config file to use the correct naming without being regenerated with a wrong one on each CLI usage?